src/Domain/CoordinateMaps/TimeDependent/Translation.hpp

```diff
@@ -174,6 +169,22 @@ class Translation {
   friend bool operator==(  // NOLINT(readability-redundant-declaration)
       const Translation<LocalDim>& lhs, const Translation<LocalDim>& rhs);
 
+  // This helper function computes the translated coordinates or frame velocity
+  // based on the option passed in, 0 for translated coordinates, and frame
+  // velocity for any other number.
+  template <typename T>
+  std::array<tt::remove_cvref_wrap_t<T>, Dim> coord_helper(
+      const std::array<T, Dim>& source_coords, double time,
+      const std::unordered_map<
+          std::string,
+          std::unique_ptr<domain::FunctionsOfTime::FunctionOfTime>>&
+          functions_of_time,
+      const double& option) const;
+
+  double root_finder(const double& target_radius,
+                     const double& squared_function_of_time,
+                     const double& root_factor) const;
```

Comment:
These `double`s don't need to be references`. They can just be `double` here and then `const double` in the cpp.